### ISSUE TITLE
[BUG] Optimized SVG is different than source Coat of arms of Makedonska Kamenica Municipality.svg

### ISSUE THREAD

**Issue by sk-** [svg/svgo#1385]
Using the (wip) integration test tool we identified an issue while optimizing the file `Coat of arms of Makedonska Kamenica Municipality.svg` with the default settings.

## SVGO version
2.1.0

## Differences

The book is much thicker and there are shapes that weren't there before.

## Sources
Original:
[source.svg.txt](https://github.com/svg/svgo/files/6054927/source.svg.txt)

Optimized:

[optimized.svg.txt](https://github.com/svg/svgo/files/6054928/optimized.svg.txt)

## Renders
Here are the two rendered images for comparison:

Original:

![source](https://user-images.githubusercontent.com/911768/109393413-3de04180-7900-11eb-969f-3e14b078f457.png)

Optimized:

![optimized](https://user-images.githubusercontent.com/911768/109393414-3f116e80-7900-11eb-90c1-5d4c8ab1b2b2.png)

Diff:

![diff](https://user-images.githubusercontent.com/911768/109393415-3faa0500-7900-11eb-9c4b-42f42e799ce9.png)

**Comment by TrySound** [comment]
You need to enable convertStyleToAttrs. I disabled it to fix a few issues but looks like this created other issues.
```
const { extendDefaultPlugins } = require('./lib/svgo-node.js');

module.exports = {
  plugins: extendDefaultPlugins([
    {
      name: 'convertStyleToAttrs',
      active: true
    }
  ])
};

```

**Comment by sk-** [comment]
Do you know which of the plugins is the one breaking it? Is there something that could be done in order to make that plugin safer?

**Comment by TrySound** [comment]
I made inline styles a deoptimisation for applyTransform to fix this problem.
https://github.com/svg/svgo/commit/79dbb4bf6e7464e946d838ade0aa23c7189762f7

**Comment by TrySound** [comment]
Fixed in 2.2.0
